rewrite-java-core/src/main/java/com/azure/recipes/v2recipes/TypeReferenceRecipe.java

```diff
@@ -0,0 +1,125 @@
+package com.azure.recipes.v2recipes;
```

Comment:
Instead of `v2recipes` we should call this `core.v2` i.e `com.azure.recipes.core.v2`

Reply:
Changed in commit [492daeb](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/492daeb48360470829cc54218911b9b3e1758ace)